### PR TITLE
feat: :sparkles: Use generic loader for Pico >= 4

### PR DESCRIPTION
**Pull request by zmerp**
Tested on Pico 4 Ultra. I need this for work

### CODE REVIEW

File: alvr/client_openxr/src/lib.rs [hunk 1]
```diff
@@ -118,7 +118,7 @@ pub fn entry_point() {
 
     let loader_suffix = match platform {
         Platform::Quest1 => "_quest1",
-        p if p.is_pico() => "_pico",
+        Platform::PicoNeo3 => "_pn3",
```
Comment (The-personified-devil) [inline alvr/client_openxr/src/lib.rs:121]: are we sure we don't want to do it for unknown too?

Comment (zmerp) [reply alvr/client_openxr/src/lib.rs:121]: yeah, thanks

Comment (The-personified-devil) [reply alvr/client_openxr/src/lib.rs:121]: There's no older pico devices that alvr runs on that we're not keeping track of?

Comment (zmerp) [reply alvr/client_openxr/src/lib.rs:121]: No others that run OpenXR

**Review by The-personified-devil — APPROVED** [review]

**Pull request merged**